fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -802,26 +782,10 @@ impl A380ElevatorFactory {
     fn new_elevator(context: &mut InitContext, id: ActuatorSide) -> ElevatorAssembly {
         let init_drooped_down = !context.is_in_flight();
 
-        let assembly_outward = Self::a380_elevator_assembly(
-            context,
-            init_drooped_down,
-            if id == ActuatorSide::Left {
-                Some(Self::LEFT_OUTWARD_PANEL_EHA_BUS)
-            } else {
-                Some(Self::RIGHT_OUTWARD_PANEL_EHA_BUS)
-            },
-            true,
-        );
-        let assembly_inward = Self::a380_elevator_assembly(
-            context,
-            init_drooped_down,
-            if id == ActuatorSide::Left {
-                Some(Self::LEFT_INWARD_PANEL_EHA_BUS)
-            } else {
-                Some(Self::RIGHT_INWARD_PANEL_EHA_BUS)
-            },
-            false,
-        );
+        let assembly_outward =
```

Comment:
Also wrong here: left inner is AC ESS, right inner is AC EHA (correct), left outer is AC EHA (correct), right outer is AC ESS again.